processor/k8sattributesprocessor/metadata.yaml

```diff
@@ -94,6 +94,10 @@ resource_attributes:
     description: Container ID. Usually a UUID, as for example used to identify Docker containers. The UUID might be abbreviated. Requires k8s.container.restart_count.
     type: string
     enabled: false
+  container.image.id:
+    description: Container image digest. Requires container.id or k8s.container.name.
```

Comment:
Can we align this description with the SemConv spec https://github.com/open-telemetry/semantic-conventions/blob/768a23c3627e4bd96146d9f713db214bee7aecf3/model/registry/container.yaml#L47?